lib/csv.rb

```diff
@@ -1342,8 +1341,24 @@ def build_header_fields_converter
       accept_nil: true,
     }
     options = @base_fields_converter_options.merge(specific_options)
+    build_converters(@initial_header_converters, options)
+  end
+
+  def writer_fields_converter
+    @writer_fields_converter ||= build_writer_fields_converter
+  end
+
+  def build_writer_fields_converter
+    specific_options = {
+      builtin_converters: Converters,
```

Comment:
This still exists.

Reply:
I am so sorry. I changed it in my last change but I didn't commit it.